doc/man/markdown/cockpit-desktop.adoc

```diff
@@ -0,0 +1,59 @@
+= cockpit-desktop(1)
+:doctype: manpage
```

Comment:
This is the doc-type to use for man pages backend in asciidoctor 
https://docs.asciidoctor.org/asciidoctor/latest/manpage-backend/